pkg/dependency/parser/conda/environment/parse.go

```diff
@@ -77,6 +78,7 @@ func (p *Parser) toPackage(dep Dependency) ftypes.Package {
 				EndLine:   dep.Line,
 			},
 		},
+		FilePath: prefix,
```

Comment:
I use `prefix` as `FilePath` in dependency parser.
This is a workaround.
Perhaps we need to add a prefix or a new field to `Package`.

Reply:
It's inefficient, but what if parsing `environment.yaml` again in `pkg/fanal/analyzer/language/conda/environment/environment.go` to just extract a prefix? Using `FilePath` is a bit hacky.

Reply:
yeah. I doubted this decision.

I didn't think about it right away, but what if we will use logic as for `package.json` file.
I mean don't implement [Parser](https://github.com/aquasecurity/trivy/blob/e9fc3e3397564512038ddeca2adce0efcb3f93c5/pkg/fanal/analyzer/language/analyze.go#L16-L19) interface:

https://github.com/aquasecurity/trivy/blob/e9fc3e3397564512038ddeca2adce0efcb3f93c5/pkg/dependency/parser/nodejs/packagejson/parse.go#L27-L41

wdyt?

Reply:
It's also okay for me.

Reply:
@DmitriyLewen Do you think you'll update it shortly?

Reply:
I want to do this today.

Reply:
@knqyf263 i updated PR (https://github.com/aquasecurity/trivy/pull/6953/commits/67daea7c8b66c589090e9bef8d9cc6a6f8c94839 + https://github.com/aquasecurity/trivy/pull/6953/commits/9200c44f449a502e5a80a4ebc781cb0eadb8e3bd)
take a look, please.